### DISCUSSION TITLE
Details list template shows "null" instead of nothing for empty managed properties after Update

### DISCUSSION THREAD

**Discussion by FloWaeger** [microsoft-search/pnp-modern-search#3723] [Q&A]
Whithin the details list template I am get the value "null" displayed when the managed property is empty instead of nothing as it was before I updated to 4.11.1

![image](https://github.com/microsoft-search/pnp-modern-search/assets/102596638/9287d993-845b-4084-a308-27d2298ea430)

The managed property I am using does not contain any value.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/102596638/fcaf41d6-1006-42aa-a802-07b0ebd74c01)

It looked much better without displaying "null".

Any ideas?

**Comment by wobba** [comment]
This should have been fixed with https://github.com/microsoft-search/pnp-modern-search/pull/3620 in the 4.11 release. But seems we need the fix in more template. @patrikhellgren is this correct?
